haas/packages/public/.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+  "python.pythonPath": "/usr/local/opt/python/bin/python3.7",
```

Comment:
This seems like it shouldn't be committed to the tree. I for one, do not have python at this location.

Reply:
Yep, good catch!